app/models/user_access.rb

```diff
@@ -67,6 +95,48 @@ def can_access_record?(resources, record)
     resources.exists?
   end
 
+  def can_create?(model, record)
+    case model
+    when :facility
+      can_create_record?(:facility, record, accessible_facility_groups(:manage))
```

Comment:
Why redirect to the `can_create_record?` methods? Why not just inline that behavior here?